.gitignore

```diff
@@ -1,5 +1,68 @@
-.oracle_jre_usage/
-.idea/
-*.pyc
-ras-collection-instrument/uploads
+# Byte-compiled / optimized / DLL files
```

Comment:
Ok, not sure I understand where this is coming from. The .gitignore is the standard template generated by GitHub for python applications, I've made three changes which are to add .idea, .build and .build test (??)